### ISSUE TITLE
Bravo's Subligar drop rate too low

### ISSUE THREAD

**Issue by alex69** [EdenServer/community#1469]
Hello,

Drop rate should be like 10% but I'm like 0/15 (and killed him lot of times in a row).
All my attempts where with my THF57/NN28 so i got TH2
I got friends like Kochie (THF75 TH3) who is 0/16+ and Buffalo 0/20+ (BLU60+).

[https://ffxiclopedia.fandom.com/wiki/Odqan](https://ffxiclopedia.fandom.com/wiki/Odqan)
Thanks

**Comment by MGramolini** [comment]
I think you are trying to make the drop rate sound worse than it is... I personally know Kochie has got at least 2 drops from this NM if not more.

**Comment by alex69** [comment]
Are you saying I'm lying ? Dude I'm camping Odqan since Saturday non stop. I talked a lot with Kochie very often these last days and again that morning and he told me he was 0/16 just ask him if you don't believe me, and ask to Buffalo that even gave it up before telling such things.

**Comment by MGramolini** [comment]
I guess Kochie must have bought and resold those multiple pairs then.  Kochie Sold multiple pairs.. I guess he never got the drop though.

**Comment by alex69** [comment]
Yup i guess so cause i checked him lot of times (he got a very impressive stuff) and I never saw im gear any bravos...

**Comment by kochiemonster** [comment]
I have had a few drop but I'm currently 0/15 or so as well. Just feels like a unlucky run.

**Comment by alex69** [comment]
Update : 
Kichie : 0/18 in a row
Me : 0/18 in a row
We got all claims

**Comment by beauxq** [comment]
Streaks are not relevant, either lucky or unlucky. Don't report how many kills since last drop. Report total number of kills and total number of drops.

**Comment by alex69** [comment]
So, as we got all claims for a week now, its like a 0/38 atm.

**Comment by beauxq** [comment]
Kochie said "I have had a few drop".
That means the total number of drops is not 0.

**Comment by alex69** [comment]
That was many weeks ago tho he didnt mentioned it. My report is 0/38 since last Saturday morning.
